packages/web/app/src/lib/preflight-sandbox/graphiql-plugin.tsx

```diff
@@ -0,0 +1,550 @@
+import { ComponentProps, Dispatch, useCallback, useEffect, useRef, useState } from 'react';
+import { clsx } from 'clsx';
+import type { editor } from 'monaco-editor';
+import { useMutation, useQuery } from 'urql';
+import { persist } from 'zustand/middleware';
+import { shallow } from 'zustand/shallow';
+import { createWithEqualityFn } from 'zustand/traditional';
```

Comment:
Lets remove these dependencies and use React.Provider if we need to teleport properties into the plugin.
I don't even understand why we even need to store this in local storage.

Reply:
for which reason? just to remove dep? less bundle size? zustand is perfectly fits in current needs
- updates localStorage for disabling preflight script or saving env state in it
- I don't use any providers (I don't need them)
- [`onModifyHeaders`](https://github.com/graphql-hive/console/blob/7cf49048996e1965e2ef4cf34218f63e3fac6fd9/packages/web/app/src/pages/target-laboratory.tsx#L247) is outside of React tree and never rerenders